### PR TITLE
Milestone 2 ; Calculator UI

### PR DESCRIPTION
**Pull request by Nemwel-Boniface**
## Hello Reviewer(s) :wave: 
In this milestone I was able to:

- [x] Use a Calculator component to parse the calculator base UI
- [x] Documented my project professionally with a descriptive README.md file
- [x] Met all the milestone project's requirements
- [x] Deployed application
- [x] No linter errors

Thank you for taking the time to review my project.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import './App.css';
 
-import Greetings from './components/greetings';
+import Calculator from './components/Calculator';
 
 function App() {
```
Comment (Mihndim2020) [inline src/App.js]: - [x] As it is a requirement for this project, all your components are supposed to `class-based`, checkout the [seventh requirement ](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md) and kindly fix that.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION


# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (Nemwel-Boniface) [reply src/App.js]: Hello @Mihndim2020 Thank you for reviewing my project.
I believe I did use class-based components. Kindly have a look at my calculator component [here](https://github.com/Nemwel-Boniface/math-magicians/blob/calculatorUI/src/components/Calculator.js).
Do correct me if I am wrong.

Comment (Mihndim2020) [reply src/App.js]: Hello @Nemwel-Boniface, please convert your `App.js` component `class-based`, presently it is functional component.

Comment (Nemwel-Boniface) [reply src/App.js]: Hello @Mihndim2020  I made the required changes.

**Review by elyor-doniyorov — APPROVED** [review]
Hi @Nemwel-Boniface, 

In my opinion, your current milestone is complete!  There is nothing else to say other than... welldone :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**